quupe-RN/js/redux/modules/Token.js

```diff
@@ -22,6 +25,7 @@ export const getError = error => ({
 
 export const deleteTheToken = () => ({
     type: DELETE_TOKEN
+<<<<<<< HEAD
```

Comment:
same thing here and as I look further more in this file, there are more merge conflicts. Please resolve them.